app/controllers/api/v1/podcasts_controller.rb

```diff
@@ -19,9 +28,19 @@ def show
         render json: podcast
     end
 
+    def destroy
+        podcast = Podcast.find(params[:id])
+        podcast.destroy
+        render json: {}, status: :no_content
+    end
+
     private
 
     def podcast_params
         params.require(:podcast).permit(:name, :url)
     end
+
+    def serialized_data(data, serializer)
+        ActiveModelSerializers::SerializableResource.new(data, each_serializer: serializer)
+    end
```

Comment:
As you have the helper method for serializer, can you change line 7 as: 
 user = serialized_data(current_user, UserSerializer)